Tests/SwiftDocCTests/Servers/DocumentationSchemeTests.swift

```diff
@@ -19,11 +19,11 @@ import XCTest
 fileprivate let baseURL = URL(string: "test://")!
 fileprivate let helloWorldHTML = "<html><header><title>Hello Title</title></header><body>Hello world</body></html>".data(using: .utf8)!
 
-class TopicRefenceSchemeTests: XCTestCase {
+class DocumentationSchemeTests: XCTestCase {
```

Comment:
Should this be called `DocumentationSchemeHandlerTests` to align with the `DocumentationSchemeHandler` type?

Reply:
Should we change the test file name too? (From XXTests to XXHandlerTests) 

🤔️

> Just follow the original convention since it not contains the handler